### ISSUE TITLE
VrVideoView on SurfaceView (Z-Order Overlay)

### ISSUE THREAD

**Issue by yooseunghun** [googlevr/gvr-android-sdk#261]
now I develop runtime application with SurfaceView. 
I'd like to use VrVideoView on SurfaceView, but there is no way to resolve Z-Order Overlay.
Now VrWidgetView (which is extended by VrVideoView) uses GLSurfaceView, so please let it(or VrVideoView) be able to set z-oder overlay ( SurfaceView.setZOrderMediaOverlay(boolean) )

**Comment by sigmaxipi** [comment]
What are you using the SurfaceView for? Changing the compositing behavior of the VrWidgetView can cause some issues when due to the lifecycle of VrWidgetView on rotations and other layout changes.

**Comment by yooseunghun** [comment]
@sigmaxipi 
I develop an application which has a display rendered by native C code thru JNI. So Now I use SurfaceView. What are the issues ? Can you let me know more details? Now I have already used VideoView(which is extended from SurfaceView) on my SurfaceView as well. But there is no issue about rotations, layout chagnes.

**Comment by sigmaxipi** [comment]
We ran into many issues related to Surface compositing in the past so this will require some more testing on multiple devices before we can determine if the proposed API works as expected. As an unsupported work around, you could directly access the underlying GLSurfaceView (via a series of getChildAt(0) calls) and try setting its Z Order directly. This hack may break if we change the layout, but it should work for you until we can determine that the Z order works as expected for all our supported devices.

**Comment by genshenx** [comment]
@sigmaxipi 
I develop a player which show the picture by native method : ANativeWindow_lock() and ANativeWindow_unlockAndPost(), I know in VR mode the display latency has been cut down to less than 20ms,  So how can I make  my display latency to about 20ms? Can you give me some advices?
Or I shoud use the other display methods? Thanks

**Comment by dav-cz** [comment]
@genshenx please open a separate issue for this. Closing this as using VrVideoView with SurfaceView is not a supported use case.
